ephios/core/templates/core/event_list.html

```diff
@@ -106,35 +107,130 @@ <h1>{% translate "Events" %}
     {% if mode == "calendar" %}
         {{ calendar }}
     {% elif mode == "day" %}
-        <table class="table">
-            <thead>
-                <tr>
-                    <td>Time</td>
-                    {% for event in events %}
-                        <td>{{ event.title }}</td>
+        <style nonce="rAnd0m">
+            .day-calendar-container {
+                grid-template-columns: {{ css_grid_columns }};
+                grid-template-rows: [head] 3em [content] 1fr;
+            }
+
+            .day-calendar-head {
+                grid-row: head;
+            }
+
+            .day-calendar-content {
+                grid-row: content;
+                position: relative
+            }
+
+            .day-calendar-content > .card {
+                margin: 0.1em;
+                position: absolute;
+            }
+
+            .day-calendar-content > .card:hover {
+                z-index: 2;
+            }
+
+            {% for column in shift_columns %}
+                .day-calendar-column-{{ column }} {
+                    grid-column: {{ column }};
+                }
+            {% endfor %}
+
+            {{ css_shift_tops }}
+        </style>
+        <div class="day-calendar-container">
+            {% for event in events %}
+                <div class="day-calendar-column-{{ event.get_canonical_slug }}-0 day-calendar-head">
```

Comment:
is there a reason why you are using the slug instead of the pk? Slugs may change, pks are fixed.

Reply:
That shouldn't matter here, because the layout is recalculated on reload anyways. Slugs were nicer to develop against because the class names etc. were human readable, but of course we can easily switch this to pks before merging if you prefer.

Reply:
Is the thumbs up a "I don't care, keep it" or a "change it before merging"?

Reply:
please change it to primary keys, that's what we are doing everywhere else...